portal/views/admin.py

```diff
@@ -45,21 +45,18 @@
 from django.contrib.auth.models import User
 from django.contrib.auth.decorators import permission_required, login_required
 from django.contrib import messages as messages
-from django_recaptcha_field import create_form_subclass_with_recaptcha
 from django.utils import timezone
 
-from recaptcha import RecaptchaClient
```

Comment:
Well spotted, these imports are indeed not needed anymore. However, it seems like you need to get rid of those same imports in the other view files: this is what causes the tests to fail.